ReactiveCocoa/RACSignal+Operations.m

```diff
@@ -270,6 +270,17 @@ - (RACSignal *)catchTo:(RACSignal *)signal {
 	}] setNameWithFormat:@"[%@] -catchTo: %@", self.name, signal];
 }
 
++ (RACSignal *)try:(id (^)(NSError **errorPtr))tryBlock {
+	NSCParameterAssert(tryBlock != NULL);
+
+	return [[RACSignal createSignal:^(id<RACSubscriber> subscriber) {
+		NSError *error;
+		id value = tryBlock(&error);
+		RACSignal *signal = (error == nil ? [RACSignal return:value] : [RACSignal error:error]);
```

Comment:
As @Coneko pointed out, this should check `value == nil`. This is the [pattern for `NSError` handling](https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/ErrorHandlingCocoa/CreateCustomizeNSError/CreateCustomizeNSError.html).

> __Important__: Success or failure is indicated by the return value of the method. Although Cocoa methods that indirectly return error objects in the Cocoa error domain are guaranteed to return such objects if the method indicates failure by directly returning `nil` or `NO`, you should always check that the return value is `nil` or `NO` before attempting to do anything with the `NSError` object.